app/src/main/res/values/dimens.xml

```diff
@@ -8,4 +8,8 @@
     <dimen name="dp_3">3dp</dimen>
     <dimen name="sp_12">12sp</dimen>
     <dimen name="dp_0">0dp</dimen>
+    <dimen name="dp_5">5dp</dimen>
+    <dimen name="dp_60">60dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_60` appears to be unused
 Unused resources make applications larger and slow down builds.